mava/components/training/advantage_estimation.py

```diff
@@ -57,7 +58,10 @@ def on_training_utility_fns(self, trainer: SystemTrainer) -> None:
         """
 
         def gae_advantages(
-            rewards: jnp.ndarray, discounts: jnp.ndarray, values: jnp.ndarray
+            rewards: jnp.ndarray,
+            discounts: jnp.ndarray,
+            values: jnp.ndarray,
+            stats: jnp.ndarray = jnp.array([0, 1, 1e-4]),
```

Comment:
I think you added docs for this below, but can you please add docstrings for this function.

Reply:
They was already a docstring but I have added more comments.